node/src/main/scala/coop/rchain/node/NodeRuntime.scala

```diff
@@ -318,18 +320,18 @@ class NodeRuntime private[node] (
         }
       } >> exit0.as(Right(()))
 
-  private def localScope(source: Metrics.Source): Task[ApplicativeLocal[Task, Trace]] =
-    TaskLocal[Trace](Trace.source(source))
+  private def localScope(source: Metrics.Source): Task[ApplicativeLocal[Task, TraceId]] =
+    TaskLocal[TraceId](UUID.randomUUID())
```

Comment:
wouldn't an atomicInteger be faster, and easier to debug? For sure it wouldn't have even the slightest risk of duplicating, which randomUUID does have